spiceaidocs/docs/cli/reference/catalogs.md

```diff
@@ -1,27 +1,31 @@
 ---
-title: "catalogs"
-sidebar_label: "catalogs"
-pagination_prev: null
-pagination_next: null
----
+ title: "catalogs"
+ sidebar_label: "catalogs"
+ pagination_prev: null
+ pagination_next: null
+ ---
+
+ List [catalogs](/components/catalogs) currently loaded by the Spice runtime.
 
-Lists [catalogs](/components/catalogs) loaded by the Spice runtime
+ ### Requirements
 
-### Usage
+ - Spice runtime must be running
+
+ ### Usage
 
 ```shell
-spice catalogs [flags]
+> spice catalogs [flags]
 ```
 
-#### Flags
+ #### Flags
 
-- `--tls-root-certificate-file`   The path to the root certificate file used to verify the Spice.ai runtime server certificate
+ - `--tls-root-certificate-file` The path to the root certificate file used to verify the Spice.ai runtime server certificate
 
-### Example
+ ### Example
 
-```shell 
->>> spice catalogs
+ ```shell
+ > spice catalogs
 
-FROM    NAME     
-spiceai spiceai 
-```
+ FROM    NAME
+ spiceai spiceai
```

Comment:
nit: Example should have >1 row, and a row that is not spiceai. See [here](https://docs.spiceai.org/components/catalogs/) for other catalogs we support.